wiki/Tournaments/OWC/1/en.md

```diff
@@ -81,39 +85,49 @@ The osu! World Cup #1 was run by various community members by distributing the m
 | ![][flag_GB] | **United Kingdom** | **Doomsday**, Natteke, aevv, Jericho2442, DiamondCrash |
 | ![][flag_US] | **United States** | **Lybydose**, Mafiamaster, Cyclone, ebacho, naptime |
 
----------------------------------------------------
+| Asia/Oceania | Europe | America |
+| ------------ | ------ | ------- |
+| ![][flag_CN] China       | ![][flag_AT] Austria | ![][flag_AR] Argentina |  
+| ![][flag_ID] Indonesia   | ![][flag_FI] Finland | ![][flag_BR] Brazil |
+| ![][flag_JP] Japan       | ![][flag_FR] France | ![][flag_CA] Canada |
+| ![][flag_KR] Korea       | ![][flag_DE] Germany | ![][flag_CL] Chile |
+| ![][flag_MO] Macau       | ![][flag_IT] Italy | ![][flag_DO] Dominican Republic |
+| ![][flag_MY] Malaysia    | ![][flag_NL] Netherlands | ![][flag_US] United States | 
+| ![][flag_NZ] New Zealand | ![][flag_PL] Poland | |
+| ![][flag_PH] Philippines | ![][flag_PT] Portugal | | 
+| ![][flag_TW] Taiwan      | ![][flag_RU] Russia | |
+| | ![][flag_ES] Spain | |
+| | ![][flag_SE] Sweden | |
+| | ![][flag_UA] Ukraine | |
+| | ![][flag_GB] United Kingdom | |
 
-![OWC #1 brackets](brackets.png "Brackets")
-
-----------------------------------------------------
+---------------------------------------------------
 
-## Mappools
+## Mappool
 
 ### Finals
 
-**Played in order**
-
-1. [Glasslake - Driving Lazy Bee (darrihuka) \[DaRRi MIx\]](https://osu.ppy.sh/beatmapsets/3861#osu/22759)
-2. [YUI - Sea (VanMoNky) \[Senior\]](https://osu.ppy.sh/beatmapsets/13021#osu/48422)
-3. [Suzaku - Anisakis -somatic mutation type "Forza" (tsukamaete) \[Another\]](https://osu.ppy.sh/beatmapsets/15579#osu/56347)
-4. [IOSYS - Danzai Yamaxanadu (Zekira) \[Eternal Damnation\]](https://osu.ppy.sh/beatmapsets/13654#osu/50354)
-5. [DJ YOSHITAKA - Bloody Tears(IIDX EDITION) (Gabi) \[Insane\]](https://osu.ppy.sh/beatmapsets/6598#osu/29731)
-6. [Betwixt & Between - I think we can go to the moon (AngelHoney) \[Another\]](https://osu.ppy.sh/beatmapsets/13728#osu/50658)
-
-**Tiebreaker (3-3)** [Renard - Banned Forever (Blue Dragon) \[Nogard\]](https://osu.ppy.sh/beatmapsets/16349#osu/64267)
+- NoMod (played in order)
```

Comment:
might as well remove the `(played in order)` part if the numbered list is converted to a bulleted one
(and for the other one below too)